libgalois/src/analytics/bfs/bfs.cpp

```diff
@@ -431,33 +474,13 @@ BfsImpl(
 
   // TODO(lhc): due to lack of in-edge iteration, manually creates a transposed graph
   const katana::GraphTopology& topology = pg->topology();
-  katana::LargeArray<uint32_t> node_data;
-  bool use_block = false;
-  if (use_block) {
-    node_data.allocateBlocked(topology.num_nodes());
-  } else {
-    node_data.allocateInterleaved(topology.num_nodes());
-  }
   auto transpose_graph = katana::CreateTransposeGraphTopology(topology);
 
   katana::do_all(katana::iterate(graph.begin(), graph.end()), [&](auto n) {
     graph.GetData<BfsNodeParent>(n) = BfsImplementation::kDistanceInfinity;
```

Comment:
Drop this do_all because you are setting this after execution. It does not need to be initialized.